chainloader/Makefile

```diff
@@ -92,7 +92,7 @@ flash: flash-$(HOST_OS)
 
 flash-Linux: $(IMAGE_NAME).img
 	mkdir -p $(MNT)
-	mount $(MNT)
+	@mount $(MNT) && echo "mounted SD card to $(MNT)" || echo "mount failed, expecting card to be already mounted, continuing..."
```

Comment:
```suggestion
	@mount $(MNT) && echo "mounted SD card to $(MNT)" || echo "Warning: SD card is already mounted"
```

Reply:
alternative solution committed in 8b610ef